### ISSUE TITLE
Building the docs locally: missing info in documentation

### ISSUE THREAD

**Issue by JennaVergeynst** [UGentBiomath/COVID19-Model#150]
@stijnvanhoey As indicated on the documentation website, I did `python setup.py build_sphinx` to build the docs locally. 
I had to install sphinx, sphinx_rtd_theme, numpydoc and myst_parser before it worked.
Should we either mention this on the documentation page, or rather add these packages to the environment.yml file?

* Python version: 3.8.2

**Comment by JennaVergeynst** [comment]
@stijnvanhoey @jorisvandenbossche can you have a look at this issue?

**Comment by stijnvanhoey** [comment]
@JennaVergeynst these are only enlisted in the development requirements, see https://github.com/UGentBiomath/COVID19-Model/blob/master/setup.py#L24-L28

Is is explained in the https://ugentbiomath.github.io/COVID19-Model/collaborate.html#documentation-website section:

> In case you want to build the documentation locally, make sure you have the development dependencies installed (`pip install -e ".[develop]"`) to run the sphinx build script. The build sphinx script relies on the setuptools integration of Sphinx:

with the `pip install -e ".[develop]"` installing those dependencies. Can I adjust/add something to make this more clear?

__note__ the quotes around `"[develop]"` are required only for the @Beramos alike zsh-shell users ;-) you can also do:

```
pip install -e .[develop]
```
(both with and without will work)

**Comment by Beramos** [comment]
Sorry 👼

Op ma 26 okt. 2020 15:21 schreef Stijn Van Hoey <notifications@github.com>:

**Comment by JennaVergeynst** [comment]
Yes if you can make this more clear, go ahead! Thanks @stijnvanhoey!
